lib/commands/submit_order/handler.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const PI = require('p-iteration')
+const _isString = require('lodash/isString')
+const _isEmpty = require('lodash/isEmpty')
+const _isArray = require('lodash/isArray')
+const script = require('bitfinex-api-node/examples/rest2/submit_order')
+
+const paramsFromForm = require('./params_from_form')
+const processPiped = require('./process_piped')
+
+script.skipAutoExec()
+
+module.exports = async (argv) => {
+  const { _, market } = argv
+  const args = _.slice(1)
+
+  if (_isString(market) && !_isEmpty(market)) { // use supplied flags
+    const params = processPiped({
+      ...argv,
+      symbol: market
+    }, true)
+
+    return script.exec(params)
+  } else if (args.length <= 1) { // piped or no args (show form)
```

Comment:
`< 2` may save 1 character :-)

Reply:
I held off for awhile but in the end changed it, you were right :joy: